db/seeds.rb

```diff
@@ -5,3 +5,17 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+category1 = Category.create(name: 'Coffee')
+category2 = Category.create(name: 'Brewing')
+category3 = Category.create(name: 'Merchandise')
+
+item1 = category1.items.create(title: 'Non-fat latte with caramel drizzle', description: 'For the skinny but caffeinated', price: 1000, photo: 'http://i.huffpost.com/gen/1270227/thumbs/o-COFFEE-570.jpg?5')
+item2 = category1.items.create(title: 'Grande iced sugar-free vanilla latte with soy milk', description: 'For the healthy organic individual', price: 800, photo: 'http://i.huffpost.com/gen/1269829/thumbs/o-COFFEE-570.jpg?1')
+item3 = category1.items.create(title: 'Triple venti soy low-foam latte', description: 'Bring out the indulgent European in you', price: 1200, photo: 'https://encrypted-tbn2.gstatic.com/images?q=tbn:ANd9GcQA9Sn_r7xu2Y6NcHd88xGYQmYMlgaBiTv41H7US2bZ4ZloyvN1')
+item4 = category1.items.create(title: 'Non-fat frappuccino with extra whip cream and chocolate', description: 'YOLO', price: 1500, photo: 'https://s-media-cache-ak0.pinimg.com/236x/c3/b8/fb/c3b8fbaf949af7b2de462c329dc3a5be.jpg')
```

Comment:
Useless assignment to variable - `item4`.<br>Line is too long. [228/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.